iothub/service/src/Query/IQuery.cs

```diff
@@ -52,17 +52,17 @@ public interface IQuery
         Task<QueryResponse<DeviceJob>> GetNextAsDeviceJobAsync(QueryOptions options);
 
         /// <summary>
-        /// Retrieves the next paged result as <see cref="JobResponse"/> objects
+        /// Retrieves the next paged result as <see cref="ScheduledJob"/> objects
```

Comment:
Type reference not required as the API already refers to it.

```suggestion
        /// Retrieves the next paged result as scheduled jobs.
```

Reply:
Also below.